routes/entryRoutes.js

```diff
@@ -65,15 +71,24 @@ routes.POSTnewLongEntry = function(req, res, next) {
   })
 };
 
-routes.GETstudentEntries = function(req, res){
+routes.GETallStudentEntries = function(req, res) {
+  var studentID = req.query.studentID;
+  FormDB.find({_studentID:studentID}, function(err, entries) {
+    //TODO: err
```

Comment:
fixed

On Tue, Apr 19, 2016 at 4:49 PM Nora Mohamed <notifications@github.com>
wrote:

> In routes/entryRoutes.js
> <https://github.com/davidabrahams/Spring-Initiative/pull/122#discussion_r60308371>
> :
>
> > @@ -65,15 +71,24 @@ routes.POSTnewLongEntry = function(req, res, next) {
> >    })
> >  };
> >
> > -routes.GETstudentEntries = function(req, res){
> > +routes.GETallStudentEntries = function(req, res) {
> > +  var studentID = req.query.studentID;
> > +  FormDB.find({_studentID:studentID}, function(err, entries) {
> > +    //TODO: err
>
> add
>
> if(err) console.log('Error:', err);
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/davidabrahams/Spring-Initiative/pull/122/files/755e9ad82ee6b858878c4e57a249a75b6fd6336e#r60308371>
>